client/webserver/site/src/js/markets.js

```diff
@@ -988,6 +969,181 @@ export default class MarketsPage extends BasePage {
   }
 }
 
+/*
+ * tmplElement is a helper function for grabbing sub-elements of the market list
+ * template.
+ */
+function tmplElement (ancestor, s) {
+  return ancestor.querySelector(`[data-tmpl="${s}"]`)
+}
+
+/*
+ * MarketList respresents the list of exchanges and markets on the left side of
+ * markets view. The MarketList provides utilities for adjusting the visibility
+ * and sort order of markets.
+ */
+class MarketList {
+  constructor (col) {
+    this.column = col
```

Comment:
`col`/`column` sounds like integer values to me (as in row/column). Maybe `listElement` or `listEl`?